### PR TITLE
Simplify Name type

### PR DESCRIPTION
**Pull request by vladwix**

### CODE REVIEW

**Review by olegromashin — APPROVED** [review]

File: core/v0/api/core.json [hunk 1]
```diff
@@ -2695,38 +2695,9 @@
...
-                        "type": "string",
-                        "description": "An honorific prefix preceding a Person's name such as Dr/Mrs/Mr."
-                    },
-                    "honorific_suffix": {
-                        "type": "string",
-                        "description": "An honorific suffix preceding a Person's name such as M.D. /PhD/MSCSW."
-                    }
-                }
+                "type": "string",
+                "description": "Describes the name of a person in format: ./{honorific prefix}/{given_name}/{family_name}/{additional_name}/{call_sign}/{honorific_suffix}",
```
Comment (beckn-dev) [inline core/v0/api/core.json:2699]: This will have to wait until the review.

File: core/v0/api/core.yaml [hunk 2]
```diff
@@ -1764,30 +1764,9 @@ components:
...
-          type: string
-          description: 'A callsign, as used in broadcasting and radio communications to identify people'
-        honorific_prefix:
-          type: string
-          description: An honorific prefix preceding a Person's name such as Dr/Mrs/Mr.
-        honorific_suffix:
-          type: string
-          description: An honorific suffix preceding a Person's name such as M.D. /PhD/MSCSW.
+      type: string
+      description: 'Describes the name of a person in format: ./{honorific prefix}/{given_name}/{family_name}/{additional_name}/{call_sign}/{honorific_suffix}'
```
Comment (beckn-dev) [inline core/v0/api/core.yaml:1768]: This will have to wait until the review

Comment (beckn-dev) [reply core/v0/api/core.yaml:1768]: Can you please change the components of name in the following order?
1. given_name (required and default)
2. honorific_prefix
3. first_name
4. middle_name
5. last_name
6. honorific_suffix

Remove additional name and call sign for now

Comment (vladwix) [reply core/v0/api/core.yaml]: Done

**Pull request merged**